src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -261,6 +261,7 @@ def test_analyze_parse_child_logs(self, multihost, backupsssdconf):
         try:
             client.login()
         except SSHLoginException:
+            _, stdout = analyze(multihost, 'show --pam --child 1')
```

Comment:
Why is this being added with no similar line removed?

Reply:
The assert in the following line uses `stdout` and the variable is uninitialized at this point. I guess that the author meant to check the output of the command, as it is done elsewhere in this file, for the error. @shridhargadekar can you confirm my assumption?